src/main/java/org/opentripplanner/transit/service/DefaultTransitService.java

```diff
@@ -156,8 +156,10 @@ public Integer getServiceCodeForId(FeedScopedId id) {
   }
 
   @Override
-  public TIntSet getServiceCodesRunningForDate(LocalDate date) {
-    return transitModelIndex.getServiceCodesRunningForDate().get(date);
+  public TIntSet getServicesRunningForDate(LocalDate serviceDate) {
```

Comment:
Do this PR change the service code/id logic? Service codes and services are two different things. We can reference a service by the code (internal OTP optimization) and id (used outside OTP to reference a calendar service). When talking about the codes we try to use `code` explicit to avoid mixing it with the ID.

Reply:
This does not change the logic. I noticed that there were two functions returning the same value, so I picked the name from the function, which was used in more places. I can switch it to `getServiceCodesRunningForDate`

Reply:
I was a bit confused about the name change, I think we should keep the "Code" until we get rid of it.

Reply:
I've changed it to `getServiceCodesRunningForDate` The original change was done here https://github.com/opentripplanner/OpenTripPlanner/pull/4353/commits/e8e97daefa576f0c3d2212ec8f42b76cc0bbe524. The method including `Codes` in the name was only used in one place, while the other one is used in around ten places, as is visible in https://github.com/opentripplanner/OpenTripPlanner/pull/4353/commits/8485f05e465e4ed5ce3da69de6eaceadb6638f5e